pkg/webui/components/events/events.styl

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.header
+  position: relative
+  display: flex
+  align-items: center
+  justify-content: space-between
+
+  &-columns
+    width: 100%
+    display: flex
+    align-items: center
+    justify-content: flex-start
+    font-weight: 600
+
+    &-time
+      margin-left: 1.6rem
+
+    &-id
+      flex-basis: 20%
+      min-width: 7rem
+      margin-left: 2.8rem
+      margin-right: $cs.s
+
+  &-actions
+    position: absolute
+    right: 2rem
```

Comment:
Should rather attach right to the edge (as in `right: 0`). See also my other comment about handling the positioning here with flexbox.

Reply:
```suggestion
    right: 0
```